test/infra/infra-mysql84-binlog/bin/docker-orchestrator-post.bash

```
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+set -o pipefail
+[ -f .env ] && . .env
+
+# Ensure INFRA is set correctly
+if [ -z "${INFRA}" ] || [ "${INFRA}" == "." ]; then
+    export INFRA=$(basename $(cd $(dirname $0)/.. && pwd))
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)